test/lit/wasm-split/instrument-in-memory.wast

```diff
@@ -1,88 +1,46 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
 ;; RUN: wasm-split %s --instrument --in-memory -all -S -o - | filecheck %s
 
 ;; Check that the output round trips and validates as well
 ;; RUN: wasm-split %s --instrument --in-memory -all -g -o %t.wasm
 ;; RUN: wasm-opt -all %t.wasm -S -o -
 
 (module
+  ;; CHECK:      (import "env" "foo" (func $foo))
+
+  ;; CHECK:      (memory $0 1 1)
+  (memory $0 1 1)
   (import "env" "foo" (func $foo))
+  ;; CHECK:      (export "bar" (func $bar))
   (export "bar" (func $bar))
+  ;; CHECK:      (func $bar
+  ;; CHECK-NEXT:  (i32.atomic.store8
+  ;; CHECK-NEXT:   (i32.const 0)
+  ;; CHECK-NEXT:   (i32.const 1)
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT:  (call $foo)
+  ;; CHECK-NEXT: )
   (func $bar
     (call $foo)
   )
+  ;; CHECK:      (func $baz (param $0 i32) (result i32)
+  ;; CHECK-NEXT:  (i32.atomic.store8 offset=1
+  ;; CHECK-NEXT:   (i32.const 0)
+  ;; CHECK-NEXT:   (i32.const 1)
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT:  (local.get $0)
+  ;; CHECK-NEXT: )
   (func $baz (param i32) (result i32)
     (local.get 0)
   )
 )
 
 ;; Check that a memory has been added
-;; CHECK: (memory $0 1 1)
 
 ;; And the profiling function exported
-;; CHECK: (export "__write_profile" (func $__write_profile))
 
 ;; Check that the function instrumentation is correct
 
-;; CHECK:      (func $bar
-;; CHECK-NEXT:  (i32.atomic.store8
-;; CHECK-NEXT:   (i32.const 0)
-;; CHECK-NEXT:   (i32.const 1)
-;; CHECK-NEXT:  )
-;; CHECK-NEXT:  (call $foo)
-;; CHECK-NEXT: )
-
-;; CHECK-NEXT: (func $baz (param $0 i32) (result i32)
-;; CHECK-NEXT:  (i32.atomic.store8 offset=1
-;; CHECK-NEXT:   (i32.const 0)
-;; CHECK-NEXT:   (i32.const 1)
-;; CHECK-NEXT:  )
-;; CHECK-NEXT:  (local.get $0)
-;; CHECK-NEXT: )
 
 ;; Check that the profiling function is correct.
 
-;; CHECK:      (func $__write_profile (param $addr i32) (param $size i32) (result i32)
```

Comment:
I reverted this test file back to main, because it was manually created and just added the memory line I needed for the test to pass. We're all good now 👍🏾